src/gsi/m_obsNodeTypeManager.F90

```diff
@@ -125,7 +128,7 @@ module m_obsNodeTypeManager
 
   public:: iobsNode_light
   public:: iobsNode_dbz
-  public:: iobsNode_fed
+  public:: iobsNode_gnssrspd
```

Comment:
I saw all the iobsNode_fed were removed from the code base. is the iobsNode_gnssrspd supposed to replace the iobsNode_fed?

Reply:
This was a merge conflict. All "fed" instances were added to m_obsNodeTypeManager.F90.

Reply:
Thanks

Reply:
Thank you @xincjin-NOAA!